src/P3Scheme.jl

```diff
@@ -451,10 +415,447 @@ function distribution_parameter_solver(
             RS.SecantMethod(min, max),
             RS.CompactSolution(),
             RS.RelativeSolutionTolerance(eps(FT)),
-            10,
+            5,
         ).root
 
     return (; λ = exp(x), N_0 = DSD_N₀(p3, N, exp(x)))
 end
 
+"""
+    terminal_velocity_mass(p3, Chen2022, q, N, ρ_r, F_r)
+
```

Comment:
There is a lot of repetition here. I'll try to combine those two functions into one